model/src/main/resources/db-update/20060-from-2.5.0-to-2.6.0.sql

```diff
@@ -70,41 +70,41 @@ BEGIN;
 --------------------------------------------------------------------------------
 -- DeliverableAnalyzerReport
 --------------------------------------------------------------------------------
-CREATE TABLE delan_report
+CREATE TABLE deliverableanalyzerreport
     (
         operation_id        BIGINT                  NOT NULL,
         labels              TEXT,
         PRIMARY KEY (operation_id)
     );
-    ALTER TABLE delan_report
-        ADD CONSTRAINT fk_delanreport_operation
+    ALTER TABLE deliverableanalyzerreport
+        ADD CONSTRAINT fk_deliverableanalyzerreport_operation
             FOREIGN KEY (operation_id)
             REFERENCES operation(id);
 
 --------------------------------------------------------------------------------
 -- DeliverableArtifact
 --------------------------------------------------------------------------------
-CREATE TABLE deliverable_artifact
+CREATE TABLE deliverableartifact
     (
         report_id           BIGINT                  NOT NULL,
         artifact_id         INTEGER                 NOT NULL,
         built_from_source   BOOLEAN                 NOT NULL,
```

Comment:
should be `builtfromsource`. Similarly for other columns. (foreign keys are `*_id`)